packager/VisibilityChecker.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PACKAGER_VISIBILITY_CHECKER_H
+#define PACKAGER_VISIBILITY_CHECKER_H
+
+#include "ast/ast.h"
+#include "common/concurrency/WorkerPool.h"
+
+namespace sorbet::packager {
+
+class VisibilityChecker final {
+    ~VisibilityChecker() = default;
```

Comment:
ooc, what does this do?

Reply:
Makes it impossible to instantiate the class.